easybuild/easyconfigs/g/gompi/gompi-1.5.16.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '1.5.16'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+ including OpenMPI for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compname = 'GCC'
+compver = '4.8.3'
+comp = (compname, compver)
+
+mpilib = 'OpenMPI'
+mpiver = '1.6.5'
+
+# compiler toolchain dependencies
+dependencies = [
+    comp,
+    (mpilib, mpiver, '', comp),
```

Comment:
replace this with:

```python
    ('OpenMPI', '1.6.5', '', comp),
```

no need for introducing `mpilib` and `mpiver`, since they're only used in one place